docs/reference/end_to_end_planning.md

```diff
@@ -0,0 +1,157 @@
+# end_to_end_planning module
```

Comment:
for consistency with the rest, the file should be named: `end-to-end-planning.md`

Reply:
done